bin/check-jenkins-health.rb

```diff
@@ -70,7 +77,14 @@ class JenkinsMetricsHealthChecker < Sensu::Plugin::Check::CLI
 
   def run
     https ||= config[:https] ? 'https' : 'http'
-    r = RestClient::Resource.new("#{https}://#{config[:server]}:#{config[:port]}#{config[:uri]}", timeout: config[:timeout]).get
+    testurl = "#{https}://#{config[:server]}:#{config[:port]}#{config[:uri]}"
+
+    if config[:https] && config[:insecure]
```

Comment:
This will only then do insecure ssl I think we have 3 conditions:
```ruby
r = if config[:https] && config[:insecure]
       RestClient::Resource.new(testurl, timeout: config[:timeout], verify_ssl: false).get
    elsif config[:https]
       RestClient::Resource.new(testurl, timeout: config[:timeout], verify_ssl: true).get
    else
       RestClient::Resource.new(testurl, timeout: config[:timeout]).get
    end
```